src/Components/Organization/Organization.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import Divider from '@mui/material/Divider';
+import { ImageListItem, Typography } from '@mui/material';
+import { Card } from './Card';
+
+export default function Organization({
```

Comment:
to follow the same logic of the rest of the app use 
```
const Organization = ({id}) => { ... }
export default Organization

```

Reply:
Also Organization is a little generic too because there will be a lot of things about the Organization. So to choose a good name you should think on, what part of the organization this component is for? To edit it? To show it data? To create an organization?
If its to show it data then the name should be something like OrganizationData or OrganizationInfo